src/BBT.StrategyPattern/Factory/IStrategyLocator.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright © BBT Software AG. All rights reserved.
+
+namespace BBT.StrategyPattern.Factory
```

Comment:
Change namespace to `BBT.StrategyPattern`. For the consumer it's much more convenient to have all classes in a single namespace instead of requiring to import 3 namespaces.